### PR TITLE
New storm module

### PR DESCRIPTION
**Pull request by silvermang**
Attempted to update storm.py to read imd, but was only partially successful. Updated storm.py to be able to read tcvitals. Updated storm_test.py to reflect this. Updated geoclaw/tests/data to include a tcvitals.txt file and a tcvitals_geoclaw.txt file.

### CODE REVIEW

File: src/python/geoclaw/surge/storm.py [hunk 1]
```diff
@@ -482,10 +494,72 @@ def read_imd(self, path, verbose=False):
 
         Return ValueError if format incorrect or if file not IMD.
         """
-        raise NotImplementedError(("Reading in IMD files is not implemented ",
-                                   "yet but is planned for a future release."))
+
```
[the remark this answers is not in the document]
Comment (silvermang) [reply src/python/geoclaw/surge/storm.py]: yeah I'll delete it

[the remark this answers is not in the document]
Comment (silvermang) [reply src/python/geoclaw/surge/storm.py]: wait which code are you referring to? The "raise NotImplementedError"?

[the remark this answers is not in the document]
Comment (mandli) [reply src/python/geoclaw/surge/storm.py]: The previous IMD code was taken out and replaced with the `NotImplementedError` so as not to confuse things especially as we now know how difficult that is going to be to implement.

**Comment by mandli** [comment]
I am going to go ahead and merge this in so it can be included in the upcoming release but I will go in and make the mentioned changes.

**Pull request merged**